### DISCUSSION TITLE
请问词频是如何记录的？是否可以根据不同的输入字符分别记录词频？

### DISCUSSION THREAD

**Discussion by BettyJJ** [rime/home#747] [Q&A]
比如说，z 和 zai 都可以输入“在”字，那这个词频是不是记录在“在”字上面，不区分是用 z 还是 zai 输入的？
有此问题是因为，某几组同音字都非常常用，比如“在”和“再”，两个字都很常用，在候选词里的顺序经常自动变化，一会这个第一那个第二，一会这个第二那个第一，每次都得集中注意力选择。
我之前用别的输入法，可以有这么一种效果：按 z 可以输入“在”，按 zai 可以输入“再”，两个词频互不干扰，顺序固定，这样就不用费劲选择了。
现在用 Rime，如果按 z 输入了“在”，按 zai 的时候“在”也会跑到第一位，按 zai 输入了“再”，按 z 时“再”又跑到第一位了，导致还是得每次都选择。
所以想问下，有没有设置可以达到上面说的别的输入法那种效果？即按 z 永远固定“在”在第一位，按 zai 永远固定“再”在第一位。
或者退而求其次，让“在”永远在第一位，“再”永远在第二位，不参与调频，但其他字词正常调频。
Rime 是否可以做到？
感谢

**Comment by lotem** [comment]
用戶詞典記錄上屏字和該字在詞典裏標註的拼音，​不區分輸入的形式。

固定重碼字的順序，配置比較複雜，請參考以往的討論：
https://github.com/rime/home/issues/9
https://github.com/rime/librime/issues/52
https://github.com/rime/librime/issues/184

**Reply by BettyJJ** [reply]
感谢回复。我先用给第一个字设定一个超大的词频数字的方法试试，用段时间看看效果

**Comment by functoreality** [comment]
[设定项详解](https://github.com/LEOYoon-Tsaw/Rime_collections/blob/master/Rime_description.md) 里给出了设置用户短语的方法（文内搜索 `custom_phrase`），这些用户短语出现的优先级很高，因此可以用于设置置顶词。比如在 `custom_phrase.txt` 里写
```
在    z    2
再    z    1
再    zai    1
```
在我这里可以固定词序，无需修改固态词典。
不过我倒是挺奇怪，这种并不复杂的设置方法为什么在教程里很少提及。
